### DISCUSSION TITLE
Can we have more frequent releases? @kyujin-cho

### DISCUSSION THREAD

**Discussion by MohabMohie** [kyujin-cho/pixel-volte-patch#227] [General]
@kyujin-cho is it possible to enable the ci pipelines so that we can have more frequent releases with new main branch merges?

**Comment by fillwithjoy1** [comment]
I don't think it's necessary + he has to update the Play Store description with the new updates so it's just easier to stack multiple commits and then push it all in a single release. Also an app like this doesn't require that many updates in a year. Just my thoughts
